### ISSUE TITLE
Maintainability is reported above 100

### ISSUE THREAD

**Issue by tandrewnichols** [es-analysis/plato#116]
I mean, I know we're pretty good, but . . . 114 out of a possible 100 seems a little too good. See image.

![screen shot 2014-10-30 at 10 22 49 am](https://cloud.githubusercontent.com/assets/2755760/4845064/8925725c-6041-11e4-9360-cd38b648e400.png)

We're running this via grunt-plato, using the following configuration:

```js
  options: {
    complexity: {
      forin: true,
      trycatch: true
    }
  },
  server: {
    files: {
      '<%= paths.reports %>/plato/server': '<%= files.js.server %>'
    }
  }
```

`<%= files.js.server %>` is just everything in a "server" directory that ends in `.js`, except stuff in public (which is minified) and instrumented (which is instrumented for code coverage).

**Comment by jsoverson** [comment]
Should have been fixed with the escomplex update some time ago.

**Comment by kevde** [comment]
Hi it still exists with reconfiguring with complexity options.
